### ISSUE TITLE
Why do we need the COCO dataset?

### ISSUE THREAD

**Issue by sckieer** [hzy46/fast-neural-style-tensorflow#15]
I'm new to VGG, so I'm a little confused about this:

> To train a model from scratch, you should first download VGG16 model from Tensorflow Slim. Extract the file vgg_16.ckpt. 
> Then download the COCO dataset. Please unzip it, and you will have a folder named "train2014" with many raw images in it. Then create a symbol link to it:
> Train the model of "wave"

While you've downloaded the vgg_16.ckpt, so u can use it to gennerate the style gram with the style_picture???So why do we need the COCO dataset and what is vgg doing here` _, endpoints_dict = network_fn(tf.concat([processed_generated, processed_images], 0), spatial_squeeze=False)`

Thx

**Comment by 625673575** [comment]
I also had the question about it,the other neural style implementations seems don't require this irrelated dataset ,and This is the slowest solution on training operation

**Comment by elleryqueenhomels** [comment]
@sckieer @625673575 
COCO dataset is used to train the 'Image Transform Net'.
The VGG-16 is pre-trained on the ImageNet dataset. We use the pre-trained VGG-16 to define the perceptual losses (feature reconstruction loss & style reconstruction loss). VGG-16 is used only on the training process, and it is fixed.
